sql/database.sql

```diff
@@ -1831,6 +1831,8 @@ CREATE TABLE `form_encounter` (
   `shift` varchar(31) NOT NULL DEFAULT '',
   `voucher_number` varchar(255) NOT NULL DEFAULT '' COMMENT 'also called referral number',
   `discharge_disposition` varchar(100) NULL DEFAULT NULL,
+  `encounter_type_code` VARCHAR(31) NULL DEFAULT NULL COMMENT 'not all types are categories',
+  `encounter_type_description` TEXT NULL,
```

Comment:
@kchapple 
Note that I've added the encounter type code where they will be needed when populating patient models. Cypress relies heavy on these codes as they don't populate diagnosis in template that I've seen. However, QRDA IG allows but I have covered in import where diagnosis becomes an attached issue/problem to encounter.